docs/reference-architectures/sap/index.md

```diff
@@ -0,0 +1,418 @@
+---
+title: Deploy SAP NetWeaver and SAP HANA on Azure
+description:  This Microsoft Azure reference architecture shows a set of proven practices for running SAP HANA in a high availability environment on Azure.
+author: njray
+ms.date: 6/08/17
+---
+
+# Deploy SAP NetWeaver and SAP HANA on Azure
+
+This reference architecture shows a set of proven practices for running SAP HANA in a high availability environment on Azure. 
+
+![0][0]
+Figure 1. SAP HANA architecture using Microsoft Azure.
+
+\* For details about SMLG and Azure Load Balancer, see the “Load balancer” item
+in the following “Architecture” section.
+
+> [!NOTE]
+-   SAP HANA is certified for production OLAP workloads on Azure GS5-series
+    virtual machines. This reference architecture is for Azure virtual machines
+    in the G-series and M-series and differs from [SAP HANA on Azure Large Instances][azure-large-instances].
```

Comment:
What is "SAP HANA on Azure Large Instances" - and when should I use this RA instead of that?